### PR TITLE
feat: integrate `@rhds/elements/rh-footer`

### PR DESCRIPTION
**Pull request by bennypowers**
See https://issues.redhat.com/browse/RHCLOUD-32579

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -10,7 +10,16 @@ module.exports = {
...
     url: 'https://test.com',
   },
-  transformIgnorePatterns: ['<rootDir>/node_modules/(?!@patternfly/react-tokens/dist/esm|@patternfly/react-icons/dist/esm).+(js|jsx)$'],
+  transformIgnorePatterns: [
+    `<rootDir>/node_modules/(?!${[
+      '@patternfly/react-tokens/dist/esm',
+      '@patternfly/react-icons/dist/esm',
+      '@patternfly/(elements|pfe-core)/.*',
+      '@rhds/elements/.*',
+      '@rhds/tokens/.*',
```
Comment (bennypowers) [inline jest.config.js:19]:
```suggestion
      '@rhds/icons/.*',
      '@rhds/tokens/.*',
```

**Comment by Hyperkid123** [comment]
@bennypowers the install job fails with:
```
npm error code 127
npm error path /Users/martin/insights/insights-chrome/node_modules/@rhds/elements
npm error command failed
npm error command sh -c patch-package
npm error sh: patch-package: command not found
```

Is there some missing script in the elements package? Seems to be called by the post-install hook:
```
301 info run @rhds/elements@2.0.0 postinstall node_modules/@rhds/elements patch-package
302 info run core-js-pure@3.35.0 postinstall node_modules/core-js-pure node -e "try{require('./postinstall')}catch(e){}"
303 info run @rhds/elements@2.0.0 postinstall { code: 127, signal: null }
```

**Comment by bennypowers** [comment]
yes i think we can try this with [2.0.1](https://github.com/RedHat-UX/red-hat-design-system/issues/1836)

**Comment by bennypowers** [comment]
```
    /home/travis/build/RedHatInsights/insights-chrome/node_modules/@rhds/elements/elements/rh-icon/rh-icon.js:74

        await Promise.resolve().then(()=>/*#__PURE__*/ _interop_require_wildcard(require("./ssr.js")));

        ^^^^^

    SyntaxError: await is only valid in async functions and the top level bodies of modules
```

untransformed source is here: https://unpkg.com/browse/@rhds/elements@2.0.1/elements/rh-icon/rh-icon.js

this error is unfortunate, as it runs just fine both in node and browser :/

**Comment by Hyperkid123** [comment]
OK, I can now run it. I'll deal with the TS issues. There are some slight styling tweaks we should make. The page looks a bit goofy when the UI is not initialized. I'll add a couple of commits to this PR if you don't mind.

**Comment by Hyperkid123** [comment]
@bennypowers I think it works now

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**